client/src/app/config.ts

```diff
@@ -1,6 +1,6 @@
-const backendPort = "5000";
+const backendPort = "5200";
```

Comment:
@sis0k0 why is the backend port changed? Should be 5000, right? Per: https://github.com/mongodb-developer/library-management-system/blob/main/server/.env

Reply:
Good catch! I don't recall why I changed it. I'll revert this.

Reply:
Done!